lib/Text/PRN/Slurp.pm

```diff
@@ -111,7 +111,6 @@ sub _from_io_handler {
 
     my $row_count = 1;
     while (my $row = <$io>) {
-        chomp $row;
```

Comment:
Otherwise the test ("t/01-load-file.t") was failing in my local environment (debian on vm). 
I am not sure if you have noticed the failure at your end. Because of chomp, the result was short by one character when comparing against the header.